Issue 2 ingest gcp from config file

galamit86 opened the pull request:
Added config parsing function, to parse toml file into dictionary, and then convert it to a named tuple, holding a named tuple (original is list) with datasets ids as strings, and a named tuple (original is dict) to hold GCP properties (currently just 'project' and 'bucket' are used, but adding others should be transparent.

@dkapitan - this feature works, and I want to merge it into the main, to be able to work on other issues. I expect this to happen often in the future, as we discussed. Would you like to be notified/assigned as reviewer for these merges?

galamit86 committed:
get gcp project and bucket from config.toml file

galamit86 committed:
remove development print statements

galamit86 committed:
convert config from dict to named tuple

galamit86 committed:
update gitignore

galamit86 committed:
fix notebook conflict

galamit86 committed:
fix git issues with config file

galamit86 committed:
fix git issue with config file

galamit86 committed:
Merge branch 'main' into issue_2_ingest_gcp_from_config_file

dkapitan reviewed (approved):
Looks good. Personally I would rename `commands.py` to `utils.py`. It's a more commonly used name for such a module. Perhaps you could refactor that for the next commit

dkapitan commented:
@galamit86 

Yay, our first PR together! Up to you whether you want to delete the branch or not.

galamit86 commented:
@dkapitan 
Haha, yess! Eventually we'll have to get that beer ;)

I changed `commands.py` to `utils.py` in main, and pushed it... Didn't mean to do it so short and dirty, but it was over before I knew what happened.... I'm still getting the hang of git and PR's. Getting better, though.

So I deleted the other braches, and will reopen a branch when I want to keep working on that issue.
